tests/integrations/rq/test_rq.py

```diff
@@ -159,18 +163,17 @@ def test_tracing_enabled(
     queue = rq.Queue(connection=FakeStrictRedis())
     worker = rq.SimpleWorker([queue], connection=queue.connection)
 
-    with start_transaction(op="rq transaction") as transaction:
-        queue.enqueue(crashing_job, foo=None)
-        worker.work(burst=True)
```

Comment:
This creates a transaction inside a transaction. This has undefined behavior so I removed it from the test.